### PR TITLE
update cmakelists of annotation tool

### PR DESCRIPTION
**Pull request by StevenPuttemans**
like suggested by @alalek in https://github.com/Itseez/opencv/pull/3654

### CODE REVIEW

**Comment by StevenPuttemans** [comment]
@alalek this is the 2.4 fix, master coming soon. Can you check it?

**Comment by alalek** [comment]
There are no `ocv_add_executable` and `ocv_target_link_libraries` in 2.4 branch. Please remove "ocv_" prefix for these 2 lines.

BTW, it is strange, buildbot ignores this problem and silently doesn't build annotation app.

**Comment by StevenPuttemans** [comment]
Will fix it!

**Comment by alalek** [comment]
It is because wrong dependencies: `opencv_imgcodecs opencv_videoio` (not exist in 2.4). Please remove them.

**Comment by StevenPuttemans** [comment]
Will do so tomorrow at work!


Verzonden vanaf Samsung Mobile.


-------- Oorspronkelijk bericht --------
Van: Alexander Alekhin
Datum:05-03-2015 17:11 (GMT+01:00)
Aan: Itseez/opencv
Cc: Steven Puttemans
Onderwerp: Re: [opencv] update cmakelists of annotation tool (#3789)


It is because wrong dependencies: opencv_imgcodecs opencv_videoio (not exist in 2.4). Please remove them.

**Comment by StevenPuttemans** [comment]
Fixed!

File: apps/annotation/CMakeLists.txt [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
 if(NOT OCV_DEPENDENCIES_FOUND)
    return()
 endif()
 
 project(annotation)
+set(the_target opencv_annotation)
 
-ocv_include_directories("${CMAKE_CURRENT_SOURCE_DIR}" "${OpenCV_SOURCE_DIR}/include/opencv")
-ocv_include_modules(${deps})
+ocv_target_include_directories(${the_target} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}" "${OpenCV_SOURCE_DIR}/include/opencv")
```
Comment (StevenPuttemans) [inline apps/annotation/CMakeLists.txt]: @alalek it seems that these ocv lines are also unrecognized by the buildbot.
Should I delete them as well?

**Comment by StevenPuttemans** [comment]
Hmm it seems that some commands known in master are not supported for 2.4. Will try to get this fixed and see where I stand then!

**Comment by StevenPuttemans** [comment]
This is now completely uniform to traincascade CMakelists in 2.4 branch but it still fails -_- lets find out why

**Comment by StevenPuttemans** [comment]
Supplied update, it seems that the CMakelists for 2.4 was pretty well formed, just made some uniform naming with other apps :) This should build fine now!

**Comment by alalek** [comment]
Looks good, :+1:

**Comment by StevenPuttemans** [comment]
:+1:

**Pull request merged**